frontend/ui/widget/inputtext.lua

```diff
@@ -120,24 +120,27 @@ if Device:isTouchDevice() or Device:hasDPad() then
             if self.parent.onSwitchFocus then
                 self.parent:onSwitchFocus(self)
             end
-            if #self.charlist == 0 then return end -- Avoid cursor moving within a hint.
-            local textwidget_offset = self.margin + self.bordersize + self.padding
-            local x = ges.pos.x - self._frame_textwidget.dimen.x - textwidget_offset
-            local y = ges.pos.y - self._frame_textwidget.dimen.y - textwidget_offset
-            self.text_widget:moveCursorToXY(x, y, true) -- restrict_to_view=true
-            self.charpos, self.top_line_num = self.text_widget:getCharPos()
+            if #self.charlist > 0 then -- Avoid cursor moving within a hint.
```

Comment:
Is this change just for symmetry with the next one?
Otherwise, shorter equivalent seems to be:
```diff
-            if #self.charlist == 0 then return end -- Avoid cursor moving within a hint.
+            if #self.charlist == 0 then return true end -- Avoid cursor moving within a hint.
```

Reply:
> Is this change just for symmetry with the next one?

Sure, consistency :)